### PR TITLE
Fix bug where strokes don't scale

### PR DESCRIPTION
**Pull request by rsheeter**
As reported in https://github.com/googlefonts/nanoemoji/issues/157

### CODE REVIEW

File: src/picosvg/svg.py [hunk 1]
```diff
@@ -371,6 +371,7 @@ def _inherit_nondefault_overflow(attrib, child, attr_name):
             "fill": _inherit_copy,
             "fill-rule": _inherit_copy,
             "style": _inherit_copy,
+            "transform": _inherit_copy,
```
Comment (anthrotype) [inline src/picosvg/svg.py:374]: it's not sufficient to inherit and copy the transform attribute from parent group element to child element like you do here; we need an `_inherit_matrix_multiply` that calls `Affine2D.product(child_transform, group_transform)`, similar to how we do inside `apply_transforms`.

Comment (anthrotype) [reply src/picosvg/svg.py]: I pushed bc4fbf8 to fix this, PTAL

Comment (rsheeter) [reply src/picosvg/svg.py]: Oops, that was ... let's politely say silly. Ty for the fix :)

File: src/picosvg/svg_transform.py [hunk 2]
```diff
@@ -65,6 +65,9 @@ def degenerate():
     def fromstring(raw_transform):
         return parse_svg_transform(raw_transform)
 
+    def tostring(self):
+        return f'matrix({" ".join(str(v) for v in self)})'
```
Comment (rsheeter) [inline src/picosvg/svg_transform.py:69]: hm. @anthrotype I wonder if this should be ntos [from svg_meta] not just str

Comment (anthrotype) [reply src/picosvg/svg_transform.py]: good point, done

**Pull request merged**